scripts/models/color-token/color-token.js

```diff
@@ -0,0 +1,197 @@
+   *
+   * @param {DTO} dto
+   * @param {ColorToken.Colors} dto.source - ColorToken Colors to merge into the destination ColorToken Colors.
+   * @param {ColorToken.Colors} dto.destination - ColorToken Colors to be receiving a merge.
+   * @returns {ColorToken.Colors} - Merged ColorToken Colors.
+   */
+  static mergeColors({ source, destination }) {
+    return Object.entries(source).reduce(
+      (final, [color, value]) => ({
+        ...final,
+        [color]: ColorToken.mergeGrades({ source: value, destination: destination[color] }),
+      }),
+      { ...destination }
+    );
+  }
+
+  /**
+   * Merge a ColorToken into an existing ColorToken.
+   *
+   * @param {DTO} dto
+   * @param {ColorToken} dto.source - ColorToken to merge into the destination ColorToken.
+   * @param {ColorToken} dto.destination - ColorToken to be receiving a merge.
+   * @returns {ColorToken} - Merged ColorToken.
+   */
+  static merge({ source, destination }) {
+    if (source.category !== Token.CONSTANTS.CATEGORIES.COLOR || destination.category !== Token.CONSTANTS.CATEGORIES.COLOR) {
+      throw new Error(`models.ColorToken.merge() :: cannot merge non-ColorToken tokens`);
+    } else if (source.format !== destination.format) {
+      throw new Error(`models.ColorToken.merge() :: cannot merge non-matching formatted tokens`);
+    } else if (source.format !== Token.CONSTANTS.TOKEN_FORMATS.STANDARD) {
+      throw new Error(`models.ColorToken.merge() :: source token format must be "${Token.CONSTANTS.TOKEN_FORMATS.STANDARD}`);
+    } else if (destination.format !== Token.CONSTANTS.TOKEN_FORMATS.STANDARD) {
+      throw new Error(`models.ColorToken.merge() :: destination token format must be "${Token.CONSTANTS.TOKEN_FORMATS.STANDARD}`);
+    }
+
+    const data = ColorToken.mergeColors({
+      source: source.data,
+      destination: destination.data,
+    });
+
+    return new ColorToken({
+      format: Token.CONSTANTS.TOKEN_FORMATS.STANDARD,
+      category: Token.CONSTANTS.CATEGORIES.COLOR,
+      data: data,
+    }).normalize();
+  }
+
+  /**
+   * Translate the format of a ColorToken's Grades.
+   *
+   * @param {DTO} dto
+   * @param {Token.Format} dto.from - The ColorToken Grades' current format.
+   * @param {Token.Format} [dto.to] - The ColorToken Grades' desired format.
+   * @param {ColorToken.Grades} dto.grades - The ColorToken Grades to be formatted.
+   * @returns {ColorToken.Grades} - Formatted ColorToken Grades.
+   */
+  static translateGrades({ from, to = Token.CONSTANTS.TOKEN_FORMATS.STANDARD, grades }) {
+    let final;
+
+    switch (from) {
+      case Token.CONSTANTS.TOKEN_FORMATS.STANDARD:
+        final = grades;
+        break;
+
+      case Token.CONSTANTS.TOKEN_FORMATS.AUTOMATED:
+        final = Object.entries(grades).reduce((formatted, [key, value]) => {
+          const mutable = { ...formatted };
+          const grade = key.split("-").pop();
+          const { r, g, b, a } = value.rgba;
```

Comment:
what happens if there is no rgba value? do u want to provide a fallback for that here?

Reply:
The exporter, `momentum-abstract`, converts all color token values to `hex24`, `rgba`, `lch`, and `cmyk`. The most reliable, and platform adopted standard, is the `rgba` value. When provided with a color token of format `automation`, this value always exists.

I can add a flow/check if you believe it's still necessary.